src/screens/UserStats/index.tsx

```diff
@@ -0,0 +1,336 @@
+              />
+            );
+          },
+        });
+      }
+    }
+  };
+
+  useEffect(() => {
+    const getProfile = async () => {
+      try {
+        const response = await profileRef.current.get();
+        const profileStats = response.data();
+
+        if (profileStats !== undefined) {
+          setProfile(profileStats);
+
+          await getImage(profileStats.avatarRef);
+        }
+      } catch (err) {
+        Alert.alert(
+          '>.<',
+          'Conteúdo indisponível, tente novamente mais tarde.',
+          [
+            {
+              text: 'Voltar',
+              onPress: () => navigation.goBack(),
+            },
+          ],
+        );
+      } finally {
+        setIsLoading(false);
+      }
+    };
+
+    getProfile();
+  }, []);
+
+  const handleInvite = async () => {
+    toast.show({
+      duration: TOAST_DURATION,
+      render: () => {
+        return (
+          <Toast
+            status="warning"
+            title="GameSaved"
+            description="Inviting member to party"
+            textColor="darkText"
+          />
+        );
+      },
+    });
+
+    try {
+      await partyRef.current.update({
+        members: firestore.FieldValue.arrayUnion(uuid),
+      });
+
+      setIsMember(true);
+
+      toast.show({
+        duration: TOAST_DURATION,
+        render: () => {
+          return (
+            <Toast
+              status="success"
+              title="GameSaved"
+              description="Member joined to the party"
+              textColor="darkText"
+            />
+          );
+        },
+      });
+    } catch (err) {
+      Alert.alert(
+        '>.<',
+        'Não foi possível concluir a operação, tente novamente mais tarde.',
+        [
+          {
+            text: 'Ok',
+          },
+        ],
+      );
+    }
+  };
+
+  const handleBan = async () => {
+    toast.show({
+      duration: TOAST_DURATION,
+      render: () => {
+        return (
+          <Toast
+            status="warning"
+            title="GameSaved"
+            description="Banning party member"
+            textColor="darkText"
+          />
+        );
+      },
+    });
+
+    try {
+      await partyRef.current.update({
+        members: firestore.FieldValue.arrayRemove(uuid),
+      });
+
+      setIsMember(false);
+
+      toast.show({
+        duration: TOAST_DURATION,
+        render: () => {
+          return (
+            <Toast
+              status="success"
+              title="GameSaved"
+              description="Member banned to the party"
+              textColor="darkText"
+            />
+          );
+        },
+      });
+    } catch (err) {
+      Alert.alert(
+        '>.<',
+        'Não foi possível concluir a operação, tente novamente mais tarde.',
+        [
+          {
+            text: 'Ok',
+          },
+        ],
+      );
+    }
+  };
+
+  const handleMember = async () => {
+    isMember ? handleBan() : handleInvite();
+  };
```

Comment:
why are you using async function if you are not returning a promise?

Reply:
I forgotten to put await before methods